modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -15,11 +15,11 @@ Before you can add compute machines that use Red Hat Enterprise Linux as the ope
 
 .Procedure
 
-. Ensure that the `kubeconfig` file for the cluster and the installation program that you used to install the cluster are on the machine. One way to accomplish this is to use the same machine that you used to install the cluster.
+. Ensure that the `kubeconfig` file for the cluster and the installation program that you used to install the cluster are on the {op-system-base} 7 machine. One way to accomplish this is to use the same machine that you used to install the cluster.
```

Comment:
Saw another PR yesterday that had it as `kubeConfig`. Should this be `kubeConfig` or `kubeconfig`?

Reply:
I believe there is a `kubeConfig` property that can be configured (e.g., [here](https://docs.openshift.com/container-platform/4.8/rest_api/config_apis/authentication-config-openshift-io-v1.html)). But in terms of the actual file, it's `kubeconfig` AFAIK (like [here](https://docs.openshift.com/container-platform/4.8/installing/installing_bare_metal/installing-bare-metal-network-customizations.html#cli-logging-in-kubeadmin_installing-bare-metal-network-customizations)).

Reply:
Thanks for explaining that :)